assets/js/lcp-beacon.js

```diff
@@ -68,6 +68,31 @@ function getImageUrlFromElement(element) {
 let performance_images = [];
 
 async function main() {
+	// AJAX call to check if there are any records for the current URL
+	const response = await fetch(rocket_lcp_data.ajax_url, {
+		method: "POST",
+		credentials: 'same-origin',
+		body: new URLSearchParams({
```

Comment:
We use this here, and FormData on the other AJAX request, possibly we need to choose one and use the same for both?